### ISSUE TITLE
Can't enable webhooks API

### ISSUE THREAD

**Issue by galperins4** [ArkEcosystem/core#964]
**Describe the bug**
Webhooks doesnt seen to enable even when setting enabled to True in plugins.js

**To Reproduce**
1. Go to '~/.ark/config/plugins.js
2. Edited file to set the following 'enabled: true' under @arkecosystem/core-webhooks
3. Restart relay
4. Check relay logs and test webhooks API

**Expected behavior**
Be able to use the webhooks API for events. Would check enabled webhooks via /api/webhooks endpoint. Also see webhooks as enabled when starting up relay that is in sync with blockchain

**Server (please complete the following information):**
 - OS: [Ubuntu]
 - Version [18.04]

**Comment by faustbrian** [comment]
Do you have only webhooks enabled or also the webhooks API? There are 2 different values as you can use webhooks without the API and just managed them internally on your server.

```
module.exports = {
  enabled: process.env.ARK_WEBHOOKS_ENABLED,
  database: {
    dialect: 'sqlite',
    storage: `${process.env.ARK_PATH_DATA}/database/webhooks.sqlite`,
    logging: process.env.ARK_DB_LOGGING
  },
  server: {
    enabled: process.env.ARK_WEBHOOKS_API_ENABLED,
    host: process.env.ARK_WEBHOOKS_HOST || '0.0.0.0',
    port: process.env.ARK_WEBHOOKS_PORT || 4004,
    whitelist: ['127.0.0.1', '::ffff:127.0.0.1', '192.168.*'],
    pagination: {
      limit: 100,
      include: [
        '/api/webhooks'
      ]
    }
  }
}
```

**Comment by faustbrian** [comment]
Just started a local testnet and works without any issues, your config most likely is wrong.

```js
  '@arkecosystem/core-webhooks': {
    enabled: true,
    database: {
      dialect: 'sqlite',
      storage: `${process.env.ARK_PATH_DATA}/database/${process.env.ARK_NETWORK_NAME}/webhooks.sqlite`,
      logging: process.env.ARK_DB_LOGGING
    },
    server: {
      enabled: true,
      host: process.env.ARK_WEBHOOKS_HOST || '0.0.0.0',
      port: process.env.ARK_WEBHOOKS_PORT || 4004,
      whitelist: ['127.0.0.1', '::ffff:127.0.0.1', '192.168.*']
    }
  },
```

**Comment by fix** [comment]
Likely need to add this warning in the docs ( webhooks and webhooks API should be enabled)

**Comment by galperins4** [comment]
So after enabling webhooks I tried to enable the block.forged webhook to listen to. However I'm not seeing any events being emitted and am unsure why. If I'm reading the following line correctly from managers.js in the core-webhooks/lib folder

"logger.debug(`Webhooks Job ${webhook.id} completed! Event [${webhook.data.webhook.event}] has been transmitted to [${webhook.data.webhook.target}] with a status of [${response.status}].`)"

Once I subscribe I'm assuming I should see the blocks.forged event show up in the debug log which it doesn't appear to be doing.

**Comment by fix** [comment]
Forged block event is triggered when YOU forge a block. Did you see equivalent forging message in the logs of the node?

**Comment by fix** [comment]
@galperins4 is #972 fixing your issue?

**Comment by galperins4** [comment]
It partially fixed it. The event code worked but there was an issue with the webhooks code. I will push a pull request shortly that will fix it.

**Comment by galperins4** [comment]
@fix created pull request #980 to fix this error

**Comment by galperins4** [comment]
fix has been merged so I'm closing this issue
